packages/vtex-io-logger/src/core/instrumentation/routes/get-binding-for-route.ts

```diff
@@ -19,6 +19,7 @@ export function getBindingsForRoute(ctx: ServiceContext) {
       account,
       workspace,
       traceId: tracer.traceId,
+      appId: process.env.VTEX_APP_ID ?? '',
```

Comment:
Maybe it's better to let the value be `undefined`/`null` when it was not found, then logging providers will detect that documents doesn't have the field.

```suggestion
      appId: process.env.VTEX_APP_ID,
```

Reply:
solved